plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -13,6 +13,10 @@
 #include "fu-thunderbolt-firmware.h"
 #include "fu-thunderbolt-retimer.h"
 
+#define RETIMER_REPLUG_DELAY                                                                       \
+	5000000 /*5 seconds sleep until retimer is available                                       \
+				      after nvm update*/
```

Comment:
can you just put the comment on the line before please :)

Reply:
Also, maybe s/RETIMER_REPLUG_DELAY/FU_THUNDERBOLT_RETIMER_CLEANUP_DELAY?

Reply:
FU_THUNDERBOLT_RETIMER_CLEANUP_DELAY sounds good